my-login-page/src/login.jsx

```diff
@@ -5,37 +5,31 @@ import { useEffect, useState, useRef} from 'react';
 const Login =() =>{
 
   
-
   const userRef = useRef();
-  const errRef = useRef();
+  const passwordRef = useRef();
 
   const [user , setUser] = useState('');
   const [pwd , setPwd] = useState('');
   const [errMsg , setErrMsg] = useState('');
   const [success , setSuccess] = useState(false);
 
+  
+  useEffect(() => {
+    const handleUsernameChange = () => {
+      const usernameValue = userRef.current.value.toLowerCase();
+      if (usernameValue.includes('@gmail.com')) {
+        passwordRef.current.focus();
+      }
+    };
 
-  useEffect( () => {
-    setErrMsg('');
-    console.log("empty");
-  },[]
-  )
-  useEffect( () => {
-    setErrMsg('');
-    console.log("user");
-  },[user]
-  )
+    userRef.current.addEventListener('input', handleUsernameChange);
```

Comment:
i think we can use `user` state in `handleUsernameChange` since it contains the value we are looking for WDT?